src/entity/SearchResponse.java

```diff
@@ -6,14 +6,17 @@ public class SearchResponse {
   private final String highlightedText;
 
   private final String fullText;
+  private final String authorId;
 
   private final Instant time;
 
   private final String roomID;
 
-  public SearchResponse(String highlightedText, String fullText, Instant time, String roomID) {
+  public SearchResponse(
+      String highlightedText, String fullText, String authorId, Instant time, String roomID) {
     this.highlightedText = highlightedText;
     this.fullText = fullText;
+    this.authorId = authorId;
```

Comment:
I have a general question about returning authorId. We need to store this to sort by unique ID, but in displaying search results, we'd like to display the names of users, not their ID. So Search response should contain a name filed as well, although we might need to add that later in the search interactor by making a firebase call?

Reply:
now that I think about it, the easiest way is probably to store (non-unique) name as well in elastic search, sort/filter based on authID in in the getData method, but return the name instead of the authID field in the searchResponse.